l10n_it_website_portal_fiscalcode/i18n/it.po

```diff
@@ -1,32 +0,0 @@
-# Translation of Odoo Server.
```

Comment:
Come mai le traduzioni sono state rimosse?

Reply:
pensavo di doverli togliere se servono posso riportarli

Reply:
Non è necessario rimuoverle, vedi la discussione che inizia con https://discord.com/channels/753902328494424064/822435512391303209/861889733743738890 e in pratica si conclude con:
> ...
> Quindi se non c'è rinomina (non mi ero accorto di aver beccato un caso particolare) anche secondo te va bene lasciarli?
> ...
> Certamente, non vanno rimossi. Il dubbio me lo ero posto solo nel caso particolare.

@primes2h ho riassunto un bel po' la discussione di Discord, fammi sapere se c'è qualcosa da aggiungere

Reply:
@SimoRubi direi di no, nulla da aggiungere. :+1: